content/powershell/powershell.md

```diff
@@ -0,0 +1,12 @@
+---
+Title: 'PowerShell'
+Description: 'PowerShell is a command line interface and scripting language available to Windows, Mac OS and Linux.'
+Codecademy Hub Page: 'https://www.codecademy.com/catalog/subject/developer-tools'
+CatalogContent:
+  - 'learn-powershell'
+  - 'paths/computer-science'
+---
+
+**PowerShell** is a command line interface and scripting language available to Windows, Mac OS and Linux. PowerShell was introduced to the Windows operating system in 2006 and has since become a popular tool for automating many administrative tasks.
```

Comment:
```suggestion
**PowerShell** is a command line interface and scripting language available to Windows, Mac OS, and Linux. PowerShell was introduced to the Windows operating system in 2006 and has since become a popular tool for automating many administrative tasks.
```